config/dev.exs

```diff
@@ -1 +1,8 @@
+import Config
 
+config :beacon, Beacon.Repo,
```

Comment:
Is this config really necessary? 🤔

Reply:
It is if you want to run Beacon by itself for debugging in dev (e.g. `iex -S mix`)